lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -243,6 +243,10 @@ function dosomething_signup_create($nid, $uid = NULL, $source = NULL, $timestamp
     return FALSE;
   }
 
+  if (strpos($source, 'reportback') === 0) {
```

Comment:
@joe the line above checks for stathat. https://github.com/DoSomething/phoenix/blob/a47d9650be54a127adaa5e80f297c474f431fc01/lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module#L95

got it, i forgot we kept which rb got the person to sign up